base.rb

```diff
@@ -1,12 +1,12 @@
-def do_something
+def do_something(p)
   
 end
 
-def do_something2(a, b)
+def do_something2(a, b, c, d = 3)
   
 end
 
-def do_something3(a: b)
+def do_something3(c: b)
```

Comment:
__[method_definition_validator]__
* The following keyword parameters are not expected. But received. [a] at [base.rb#L16](/pocke/sandbox2/blob/dd1038c314423c66f3606b5286cc21fe00f0c93c/base.rb#L16)
* The following keyword parameters are not expected. But received. [a] at [use.rb#L3](/pocke/sandbox2/blob/dd1038c314423c66f3606b5286cc21fe00f0c93c/use.rb#L3)